modules/scanning-images.adoc

```diff
@@ -20,6 +20,43 @@ Scanner identifies the vulnerabilities in the:
 * programming language specific dependencies
 * programming runtimes and frameworks
 
+[discrete]
+== Understanding and addressing the commonly occurring Scanner warning messages
+
+When scanning images with {rh-rhacs-first}, you might see the `CVE DATA MAY BE INACCURATE` warning message.
+Scanner displays this message when it cannot retrieve complete information about the operating system or other packages in the image.
+
+This occurs because the image is based on an operating system or package manager that Scanner does not support, or because the operating system or package manager version has reached end-of-life and no longer receives security updates.
+
+It is important to address this warning to ensure that vulnerability reports are as accurate as possible.
+If you ignore the warning, vulnerability reports become inaccurate and your cluster remains vulnerable to security threats.
+
+.Warning messages
+[%autowidth]
+|===
+| *Message* | *Description*
+
+|`Unable to retrieve the OS CVE data, only Language CVE data is available.`
+|Indicates that Scanner does not officially support the base operating system of the image; therefore, it cannot retrieve CVE data for the operating system-level packages.
+
+|`Stale OS CVE data.`
+|Indicates that the base operating system of the image has reached end-of-life, which means the vulnerability data is outdated. For example, Debian 8 and 9.
+
+For more information about the files needed to identify the components in the images, see xref:../operating/examine-images-for-vulnerabilities.adoc#examine-images-for-vulnerabilities[Examining images for vulnerabilities].
+
+
+|`Failed to get the base OS information.`
+|Indicates that the target registry is unreachable on the network. The cause could be a firewall blocking `docker.io`, or an authentication issue preventing access.
+
+To analyze the root cause, create a special registry integration for private registries or repositories to get the pod logs for {product-title-short} Central. For instructions on how to do this, see xref:../integration/integrate-with-image-registries.adoc[Integrating with image registries].
+
+|`Failed to retrieve metadata from the registry.`
+|Indicates that Scanner scanned the image, but the image is old and does not fall within the scope of Red Hat Scanner Certification. For more information, see https://redhat-connect.gitbook.io/partner-guide-red-hat-vulnerability-scanner-cert/[Partner Guide for Red Hat Vulnerability Scanner Certification].
```

Comment:
The certification is for the tools that perform scanning. However, the sentence reads that image must meet Red Hat Certification.
cc: @RTann

Reply:
I think the description used for `Failed to get the base OS information.` is the correct description for this. The description here refers to  `Image out of scope for Red Hat Vulnerability Scanner Certification.`